params/turtlebot_params.yaml

```diff
@@ -54,7 +54,7 @@ bt_navigator:
     robot_base_frame: base_link
     odom_topic: odom
     navigators: ["navigate_to_pose"]
-    default_nav_to_pose_bt_xml: "/home/$USER/c1t_ws/src/c1t_bringup/behavior_trees/navigate_through_route_to_pose.xml"
```

Comment:
Retain "$USER" for now until we find a way to more easily adjust this based on the current user. This way it is clear that the value needs to be changed. Also applicable to the other changes of "$USER"